src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -153,5 +154,8 @@
       <Name>RemoteExecutorConsoleApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
What does this 'Service' thing do? I'm not familiar with this.

Reply:
Visual studio adds this automatically, i'm not sure why it does that.

Reply:
Well, it would be best not to add random things.  So, perhaps that change should be removed.